1_SettingTheScene/DevelopmentPrinciples.md

```diff
@@ -33,11 +33,11 @@ Work follows a cycle starting at the centre and the moving around and outward fo
 
 ## Only high value dependencies
 
-The VulkanSceneGraph library only has C++17, CMake and Vulkan as external dependencies. The Vulkan C headers are used rather then the Vulkan C++ header which is unnecessary as the Vulkan scene graph provides its own encapsulation of Vulkan objects in a way that is coherent with how they are used in the scene graph.
+The VulkanSceneGraph library only has C++17, CMake and Vulkan as external dependencies. The Vulkan C headers are used rather than the Vulkan C++ header which is unnecessary as the Vulkan scene graph provides its own encapsulation of Vulkan objects in a way that is coherent with how they are used in the scene graph.
 
 For runtime shader compilation support the VulkanSceneGraph library uses the [glslang](https://github.com/KhronosGroup/glslang) library. Originally integrated as an optional external dependency, it is now built internally as a submodule. While gslang support is compiled in by default it can be toggled off by setting the CMake VSG_SUPPORTS_ShaderCompiler variable to 0 before building the source. Compiling glslang within the VulkanSceneGraph library resolved problems with inconsistent 3rd party packaging of glslang, so now users can have seamless experience across platforms.
 
-During the initial development of VulkanSceneGraph various other 3rd party dependencies, like gm and glfw, were considered for features like maths and windowing, but in each of these cases it was decided to implement the required features within the project rather than add an external dependency.  The reasons for implementing the functionality within the project:
+During the initial development of VulkanSceneGraph various other 3rd party dependencies, like gm and glfw, were considered for features like math and windowing, but in each of these cases it was decided to implement the required features within the project rather than add an external dependency.  The reasons for implementing the functionality within the project:
```

Comment:
math is an Americanism, maths is correct English in this context.

Reply:
Ah, that is the consequence of my watching of TV shows and films for the US. I agree and will try to use British variant if I spot anything new.

Reply:
The VSG itself has bit of combination of English like include/vsg/maths and American "English" like Color. The Americanism's are used when there is specific history for that term in graphics.  As a Brit I naturally will choose the original English term rather than American version, unless we have a specific precedent, but alas graphic industry ha been US dominated so quite a few of the terms an uncomfortable to use even for one so poor at spelling as me.  

The vsgTutorial is online book that a Brit (me) is writing so intend to prefer proper English terms if it's straight forward English text, but if it's an industry term of art then sticking with this will be OK even it's not found in the Oxford English Dictionary.